src/config/dynamic-theme-fixes.config

```diff
@@ -10143,6 +10143,16 @@ div.formular input[type="checkbox"] {
 
 ================================
 
+fritx.me
```

Comment:
Still - it should be before fritz.box

Reply:
Ouch. I forgot 'x' is not the "final" one LOL